source/manual/licensing.html.md

```diff
@@ -0,0 +1,111 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Licensing - supporting Licensify
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+last_reviewed_on: 2020-09-10
+review_in: 9 months
+---
+
+GOV.UK runs a Scala application ([Licensify](/apps/licensify.html)) which allows users to apply and pay for licences from local authorities and other competent authorities.
+
+## Overview
+
+There is a [PDF user manual](https://licensify-admin.publishing.service.gov.uk/assets/manual.pdf) available which includes a lot of detail about the backend (administration) facilities available through the GOV.UK Licensing service.
+
+### Applications
+
+The Licensing service consists of three separately deployed applications:
+
+- Frontend: the interface used by citizens and businesses who want to apply for a license. Allows them to download the relevant forms, submit them once completed, and also handles redirection to the relevant payment service if a fee is required.
```

Comment:
We should avoid using "Frontend" here and subsequently, since we have an app called that already!

Reply:
The Legacy Ops manual actually links to the Frontend application, so I think they're referring to the same thing 🤔 
It is confusing, however, as we don't have a "Backend" application...

Reply:
Ah, so I think that's (confusingly!) true [below](https://github.com/alphagov/govuk-developer-docs/pull/2760/files#diff-b5c81e34e93e8b8b37fae6de48881dc0R30), but here, we're referring to [these ones](https://github.com/alphagov/govuk-puppet/blob/master/hieradata_aws/common.yaml#L132-L140).

Reply:
Thanks - I've rewritten this bit, hopefully it's clearer now.